src/components/exercises/ExercisesList.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FC, useState, useEffect } from 'react';
+
```

Comment:
again, autoformat would avoid blank lines here

Reply:
It's fixed